app/src/main/scala/org/alephium/explorer/Application.scala

```diff
@@ -47,10 +48,16 @@ class Application(
     syncPeriod: Duration)(implicit system: ActorSystem, executionContext: ExecutionContext)
     extends StrictLogging {
 
+  //TOTO: Temporary placeholder for executing services returning a Future.
+  //      This will eventually be removed and the all services will get started
+  //      asynchronously via the start function.
+  def awaitService[T](f: => Future[T]): T =
+    Await.result(f, 5.seconds)
```

Comment:
Nit: I am not sure it's good to make this 5 seconds implicit

Reply:
Yep one of the next PRs will remove this `Await` all together. I will merge the changes made here onto the that PR and close this.

Reply:
Kinda need to merge this. We gonna have to put up with `Await` in our code just for next the few days. 

Trying to avoid `Await` right now is a bit early and my next PR getting big with it.